Sofa/framework/Core/src/sofa/core/objectmodel/BaseData.h

```diff
@@ -242,25 +230,18 @@ class SOFA_CORE_API BaseData : public DDGNode
     /// @{
     /// True if the value has been modified
     /// If this data is linked, the value of this data will be considered as modified
-    /// (even if the parent's value has not been modified)s
-    SOFA_ATTRIBUTE_DEPRECATED__ASPECT_EXECPARAMS()
-    bool isSet(const core::ExecParams*) const { return isSet(); }
+    /// (even if the parent's value has not been modified)
+    [[nodiscard]]
```

Comment:
Just to highlight it is not "only" removing deprecated macro

Reply:
you're right ;)